example/python/prepare.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# generate iroha lib
+cmake -DSWIG_PYTHON=ON ../..
+make _iroha
```

Comment:
rename target to `pyiroha`

Reply:
it might build java as well if -DSWIG_PYTHON=ON, so `pyiroha` is not suitable here

Reply:
irohapy should be great for python and irohajava for java